app/views/projects/show.html.erb

```diff
@@ -44,12 +47,14 @@
                 <td><%= issue.title %></td>
                 <td><%= issue.issue_type %></td>
                 <td><%= issue.priority %></td>
-                <td><%= issue.assignee.name %></td>
+                <td><%= issue.assignee ? issue.assignee.name : t('issues.not_assigned') %></td>
                 <td><%= issue.status %></td>
-                <td style="text-align: center;">
-                  <%= link_to t('shared.delete'),'#',class: 'btn btn-sm btn-danger' %>
-                  <%= link_to t('shared.edit'),'#',class: 'btn btn-sm my-primary-btn ' %>
-                  <%= link_to t('shared.show'),'#',class: 'btn btn-sm my-primary-btn ' %>
+                <td class="table-center-buttons">
+                  <% if can? [:destroy, :update], issue %>
+                    <%= link_to t('shared.delete'), project_issue_path(issue.project, issue), method: :delete, data: { confirm: t('shared.deletion_confirmation_message', resource: Issue.model_name.human.downcase) }, class: 'btn btn-sm btn-danger' %>
```

Comment:
`resource: Issue.model_name.human.downcase`
Why didn't we use label from yml here?